casepro/cases/models.py

```diff
@@ -95,7 +95,11 @@ def release(self):
         self.save(update_fields=('is_active',))
 
     def as_json(self, full=True):
-        result = {'id': self.pk, 'name': self.name}
+        result = {
+            'id': self.pk,
+            'name': self.name,
+            'org': self.org.name,
```

Comment:
Does this attribute need to outside of full flag? They seem to prefer the non-full state to be just id and name.